docs/src/guides/publishing.md

```diff
@@ -35,6 +44,8 @@ scripts: {
 +   "postinstall":   "yarn ubrn:checkout && yarn ubrn:android && yarn ubrn:ios",
```

Comment:
I think this:

```suggestion
+   "postinstall":   "yarn ubrn:checkout && yarn ubrn:android --release && yarn ubrn:ios --release",
```

will obviate the need to your new paragraph.

Reply:
Ah, great!